src/commons/utils.js

```diff
@@ -128,6 +130,24 @@ const ConfigUtils = {
     return await this.getStorageKey("passesTimeout") || DEFAULT_PASSES_TIMEOUT;
   },
 
+  async setReminder(reminder) {
```

Comment:
Why is this block duplicated with storageUtils? Can one call the other instead? Feel free to make this a follow up though.

Reply:
hehe this code is shared with the popup where we don't have storageUtils. You wrote this code in the first place in this way :)